wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Create/WireTextField.swift

```diff
@@ -67,6 +67,7 @@ class WireTextField: UITextField {
         setupPadding()
         setupTextFieldProperties()
         setupTextFieldEvents()
+        setupCustomClearButton()
```

Comment:
Are you sure we want custom behavior?
Didn't the designer say that we want native behavior and in doubt the mocks are wrong?

Reply:
The native clear button from Apple breaks accessibility in terms of color and contrast. That's why we needed to introduce a custom one.